packages/bpk-tokens/src/android/base/typography.json

```diff
@@ -179,7 +179,7 @@
       "category": "font-sizes"
     },
     "TEXT_XL_FONT_WEIGHT": {
-      "value": "{!FONT_WEIGHT_REGULAR}",
+      "value": "{!FONT_WEIGHT_MEDIUM}",
```

Comment:
Also, in iOS the font weight for `base` and `lg` are still `NORMAL`, but I assume this difference between platforms is deliberate?